server/routes/visitorRoutes.js

```diff
@@ -0,0 +1,7 @@
+const visitors = require('../controller/visitors');
+
```

Comment:
Remember you updated the folder name to `controllers`, you are still referencing as `controller` here.